.github/workflows/release.yml

```
@@ -0,0 +1,718 @@
+          TAG="${TAG:-${PACKAGE_VERSION}}"
+          RELEASE_URL="https://github.com/$REPO/releases/tag/v$TAG"
+
+          text="xrpl-py ${PACKAGE_VERSION} has been successfully released and published to pypi. Release URL: ${RELEASE_URL}"
+          text="${text//\\n/ }"
+
+          curl -sS -X POST https://slack.com/api/chat.postMessage \
+            -H "Authorization: Bearer $SLACK_TOKEN" \
+            -H "Content-Type: application/json; charset=utf-8" \
+            -d "$(jq -n --arg channel "#xrpl-py" --arg text "$text" '{channel:$channel, text:$text}')"
+
+notify-failure:
```

actionlint (1.7.10)

[error] unexpected key "notify-failure" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)